it/downloads/index.md

```diff
@@ -42,26 +40,52 @@ Se hai problemi con la compilazione di Ruby considera l'utilizzo di uno
 degli strumenti di terze parti presenti nella sezione successiva. Potrebbero
 esserti di aiuto.
 
-* **Stabile Corrente:**
-  Ruby {{ site.data.downloads.stable[0] }}
+* **Versioni stabili:**{% for version in site.data.downloads.stable %}{% assign release = site.data.releases | where: "version", version | first %}
+  * [Ruby {{ release.version }}]({{ release.url.gz }})<br>
+    sha256: {{ release.sha256.gz }}{% endfor %}
+
+{% if site.data.downloads.preview %}
+* **Versioni di preview:**{% for version in site.data.downloads.preview %}{% assign release = site.data.releases | where: "version", version | first %}
+  * [Ruby {{ release.version }}]({{ release.url.gz }})<br>
+    sha256: {{ release.sha256.gz }}{% endfor %}
+{% endif %}
+
+{% if site.data.downloads.security_maintenance %}
+* **Nella fase di manutenzione di sicurezza (sono giunti a fine vita!):**{% for version in site.data.downloads.security_maintenance %}{% assign release = site.data.releases | where: "version", version | first %}
+  * [Ruby {{ release.version }}]({{ release.url.gz }})<br>
+    sha256: {{ release.sha256.gz }}{% endfor %}
+{% endif %}
+
+{% if site.data.downloads.eol %}
+* **Non più mantenute (finevita):**{% for version in site.data.downloads.eol %}{% assign release = site.data.releases | where: "version", version | first %}
```

Comment:
We should use EOL here as well in my opinion

Reply:
Anche qui, ok 👍